tests/testthat/test-bind_into_vpts.R

```diff
@@ -1,7 +1,38 @@
 test_that("bind_into_vpts() returns error on incorrect parameters", {
+  # non vpts inputs
+  expect_error(
+    bind_into_vpts(
+      example_vpts[2:5],
+      example_vpts[1],
+      "not_a_vpts",
+      example_vpts[6:9]
+    ),
+    regexp = "requires vpts objects as input",
+    fixed = TRUE
+  )
+  # non vps inputs
+  expect_error(
+    bind_into_vpts.vp(
+      example_vpts[2:5],
+      example_vpts[1]
+    ),
+    regexp = "requires vpts objects as input",
+    fixed = TRUE
+  )
```

Comment:
For a user to encounter this message, they would have to call bind_into_vpts.vp() directly, which I don't think is expected behaviour. 

As I understand it, a user either passes a supported object, or gets a different error entirely, for example in the case of a character vector: 
```

Error in UseMethod("bind_into_vpts", x) : 
  no applicable method for 'bind_into_vpts' applied to an object of class "character"
  ```